client/src/components/scorecard_page/PredictionPanel.vue

```diff
@@ -49,17 +51,21 @@ export default defineComponent({
   },
   computed: {
     predictionString(): string {
-      if (this.showWaterSystemPrediction)
+      if (this.showWaterSystemPrediction) {
         return (
           this.formatPredictionAsLikelihood(this.percentLead) ??
           this.ScorecardSummaryMessages.NOT_ENOUGH_DATA_AVAILABLE
         );
-      if (this.showParcelPrediction)
+      }
+      if (this.showParcelPrediction) {
         return (
           this.formatPredictionAsLikelihood(this.publicLeadPercent) ??
           this.ScorecardSummaryMessages.NOT_ENOUGH_DATA_AVAILABLE
         );
-      if (this.showNoPrediction) return this.ScorecardSummaryMessages.NOT_ENOUGH_DATA_AVAILABLE;
+      }
+      if (this.showNoPrediction) {
```

Comment:
(this is outside the scope of this PR) I don't love the negative for this property, when the others are positive. What's the difference between `showNoPrediction` and `!showParcelPrediction`?

Reply:
showNoPrediction means there is neither parcel nor water system prediction